.circleci/config.base.yml

```diff
@@ -1354,3 +1356,17 @@ commands:
               exit 1
             fi
           when: always
+  install_node18_windows:
+    description: Install Node 18 on Windows
+    steps:
+      - run: choco install wget -y
+      - run:
+          command: wget https://nodejs.org/download/release/v18.15.0/node-v18.15.0-x64.msi -P C:\Users\circleci\Downloads\
+          shell: cmd.exe
+      - run: MsiExec.exe /i C:\Users\circleci\Downloads\node-v18.15.0-x64.msi /qn
+      - run:
+          command: |
+            Start-Process powershell -verb runAs -Args "-start GeneralProfile"
+            nvm install 18.15.0
+            nvm use 18.15.0
+      - run: node --version
```

Comment:
https://support.circleci.com/hc/en-us/articles/8325075309339-How-to-install-NPM-on-Windows-images